### DISCUSSION TITLE
securitySchemes: customizable default token value

### DISCUSSION THREAD

**Discussion by Theodlz** [scalar/scalar#3141] [Ideas]
Currently, let's say that I have an apiKey securityScheme provided, so that in the code examples that are auto-generated the headers have "Authorization: YOUR_TOKEN".

However, I think that a lot of APIs have their own format with a prefix, like "Authorization: bearer YOUR_TOKEN", or "Authorization: token YOUR_TOKEN".

It would be great if either in the config OR the API spec, we could provide a default value that overrides scalar's default, which is for now hardcoded to YOUR_TOKEN, or if we could specify a "prefix" of some sort.

**Comment by hanspagel** [comment]
We should probably just render `Bearer` or `SomethingCustom` as a prefix, if that’s in the OpenAPI document, wdyt?
